datafusion/core/src/physical_plan/file_format/mod.rs

```diff
@@ -297,6 +310,9 @@ impl<'a> Display for FileGroupsDisplay<'a> {
             }
             write!(f, "]")?;
         }
+        if self.0.len() > max_groups {
+            write!(f, ", ...")?;
+        }
         write!(f, "]}}")?;
         Ok(())
     }
```

Comment:
Is there any way we can write a test for this code? So we don't break it in the future?